nuxt.config.js

```diff
@@ -96,6 +96,7 @@ export default {
     extend (config, ctx) {
       const svgRule = config.module.rules.find(rule => rule.test.test('.svg'))
 
+      // TODO: what is this....
       svgRule.test = /\.(png|jpe?g|gif|webp)$/
```

Comment:
Can this be removed?

Reply:
It is  not  documented. Once it's documented the TODO  can be  removed. I add TODO's where things are unclear or not tidy.